styleguide/examples/PledgeCollectiveCard.md

```diff
@@ -0,0 +1,3 @@
+```js
+<PledgeCollectiveCard />
```

Comment:
This isn't working properly on styleguide: https://deploy-preview-1556--oc-styleguide.netlify.com/#!/PledgedCollectiveCard/0